Contacts/People/Council/council.md

```diff
@@ -25,15 +25,14 @@ GAP, and since we may not be aware of your interests,
 we encourage you not to wait to be contacted but to correspond directly
 with any of the Council members if you have information that we can use.
 
-In its role as editorial board, the GAP Council
-coordinates the process by which submitted GAP packages
-are refereed. In that sense, the Council Chair acts as editor in
-chief, and the other members function as "associate editors," after the
-fashion of some journals. It is appropriate to obtain advice on submitting
-a package from any of the Council members. To submit a package, the Chair
-of the Council should be informed (by emailing
-<council@gap-system.org>). For more details, see
-the page on [submission&nbsp;procedures]({{ site.baseurl }}/Contacts/submit.html).
+In its former role as editorial board, the GAP Council used to
+coordinate the process by which submitted GAP packages
+were refereed. This process has been abolished by unanimous agreement of
+the Council in February 2024. There are other venues for this today
+which did not exist when the GAP Council was founded, such as the
+[Journal of Open Source Software (JOSS)](https://joss.theoj.org)
+or the
+[Journal of Software for Algebra and Geometry (JSAG)](https://msp.org/jsag/).
```

Comment:
@beick suggests that we should not advertise specific journals, and proposes a different text for this section, which I in turned modified a bit:
```suggestion
were refereed. This process has been abolished by the GAP Council
in February 2024, since there are other publication methods available
today which did not exist in the past, such as special journals for
software descriptions. We still continue the list of deposited packages
and encourage all users to submit their GAP packages to this list.
```

Reply:
Looks good maybe "since there are other avenues for publishing software that did not exist in the past, such as specialist journals for research software packages"?

Reply:
yeah that's better, will adjust